filestore/path_only_handlers.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import six
+
+import logging
+
+from .retrieve import HandlerBase
+from .handlers import IntegrityError
+
+logger = logging.getLogger(__name__)
+
+
+class AreaDetectorTiffHandlerPath(HandlerBase):
+    specs = {'AD_TIFF'} | HandlerBase.specs
+
+    def __init__(self, fpath, template, filename, frame_per_point=1):
+        self._path = fpath
+        self._fpp = frame_per_point
+        self._template = template.replace('_%6.6d', '*')
+        self._filename = self._template % (self._path, filename)
+
+    def __call__(self, point_number):
+        start, stop = point_number * self._fpp, (point_number + 1) * self._fpp
+        if stop > len(self._image_sequence):
+            raise IntegrityError("Seeking Frame {0} out of {1} frames.".format(
+                stop, len(self._image_sequence)))
+        return list(self._image_sequence[start:stop])
```

Comment:
As the title says, first pass.  If you look at the `AreaDetectorTiffHandler` code that is a pims object which is based on filling out a glob.